src/tests/ftest/util/fio_test_base.py

```diff
@@ -79,31 +86,30 @@ def _create_pool(self):
         # Create a pool
         self.pool.create()
 
-    def _create_cont(self):
-        """Create a TestContainer object to be used to create container."""
-        # TO-DO: Enable container using TestContainer object,
-        # once DAOS-3355 is resolved.
-        # Get Container params
-        # self.container = TestContainer(self.pool)
-        # self.container.get_params(self)
-
-        # create container
-        # self.container.create()
-        env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
-        # command to create container of posix type
-        cmd = env + "daos cont create --pool={} --svc={} --type=POSIX".format(
-            self.pool.uuid, ":".join(
-                [str(item) for item in self.pool.svc_ranks]))
-        try:
-            container = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                         shell=True)
-            (output, err) = container.communicate()
-            self.log.info("Container created with UUID %s", output.split()[3])
+    def _create_cont(self, doas_cmd):
```

Comment:
(pylint-unused-argument) Unused argument 'doas_cmd'

Reply:
This PR is a cherry-pick from master, and when it was landed on master pylint checking was broken.  There's a subsequent PR that builds on this which fixes these issues.

Reply:
@ashleypittman, Can you paste that PR here please, thanks.

Reply:
https://github.com/daos-stack/daos/commit/cdf10cb3c25c66e0d712852d50e6a08386fc99ba